package.json

```diff
@@ -22,15 +22,15 @@
     "test": "grunt test"
   },
   "dependencies": {
-    "amqplib": "^0.5.1",
+    "amqplib": "~0.5.1",
     "async": "1.5.2",
     "body-parser": "1.15.0",
     "dateformat": "1.0.12",
-    "express": "^4.11.2",
+    "express": "~4.11.2",
     "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#master",
     "logops": "1.0.0-alpha.7",
     "mqtt": "1.7.0",
-    "request": "^2.69.0",
+    "request": "2.81.0",
```

Comment:
To be the same used at the JSON agent (see https://github.com/telefonicaid/iotagent-json/pull/295)

NTC (informative)